wrapanapi/containers/__init__.py

```diff
@@ -13,9 +14,24 @@ class ContainersResourceBase(object):
     directed to the path of the resource.
     The following parameters should be statically defined:
         * RESOURCE_TYPE: (str) the resource type name in the API
+        * KIND: (str) the resource 'Kind' property as it appear in JSON
```

Comment:
```KIND``` In most cases it looks to me like we can use the classes name, instead of restating it, is this true?